mne/report.py

```diff
@@ -716,6 +728,10 @@ class Report(object):
         timepoints t such that a <= t <= b.
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+
+    Notes
+    -----
+    To hide/show all sections in the html report, press 'h'
```

Comment:
Can you see if it's possible to get the `.. versionadded` sensibly part of this statement? Does it render okay in a parenthetical?